src/Microsoft.Health.Fhir.CosmosDb.Core/Features/Storage/StoreProcedures/StoredProcedureBase.cs

```diff
@@ -0,0 +1,43 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
```

Comment:
Simplify using statements. Not all using statements in this block are required.

Reply:
done